tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -113,16 +142,40 @@ function inlineNested1()
 
 function inlineMultipleStatements()
 {
-    $a = 1;
-     ?>
+    $a = 1; ?>
     <span>Hello World!</span>
-    <?php
-    $b = 1;
-    ?>
+    <?php $b = 1; ?>
     <span>Hello World!</span>
     <?php $c = 1;
 }
 
+function test()
+{
+    ?>
+  <span>Hello World!</span>
+  <?php $a = 1;
```

Comment:
should this be on a new line?

Reply:
@mgrip should be, but difficult, work on this right now